src/ts/components/errormessageoverlay.ts

```diff
@@ -95,38 +99,39 @@ export class ErrorMessageOverlay extends Container<ErrorMessageOverlayConfig> {
     }, this.config);
   }
 
-  configure(player: PlayerAPI, uimanager: UIInstanceManager): void {
+  configure(player: PlayerAPI | MobileV3PlayerAPI, uimanager: UIInstanceManager): void {
```

Comment:
I'm not super into the UI, but I think we might consider a different approach here. 

Passing in 2 different APIs and handling the logic inside does not feel like a good approach to me. 

Isn't there some mechanic already in the UI where we can apply different facets or something similar @stonko1994 ? 

I think this will not be the only place where we have to change something in the UI because of the different mobile API.

Reply:
> I think this will not be the only place where we have to change something in the UI because of the different mobile API.

That's true.

> where we can apply different facets

We have the different UI-variants, yes 🤔 But there is nothing on component level. 

> detect where we are on, and then instantiate specific components based on that

I like the idea however not sure if in the scope of this change now. (Maybe let's discuss this offline in more detail 🤔)

Reply:
I think we can just remove it... we have a type guard anyway in this function that will later check for the `PlayerEvent` and typecast it to `MobileV3PlayerAPI`.

Reply:
Well I don't think we can just remove it. Can u at least explore the option of having a `mobile-errormessageoverlay` and a `weberrormessageoverlay`? I'd be interested in how that could look like. If it is too much effort we still can say out of scope and plan it  properly.